docs/manpages/rec_control.1.md

```diff
@@ -47,7 +47,8 @@ add-nta *DOMAIN* [*REASON*]
 :    Add a Negative Trust Anchor for *DOMAIN*, suffixed optionally with *REASON*.
 
 add-ta *DOMAIN* *DSRECORD*
-:    Add a Trust Anchor for *DOMAIN* with DS record data *DSRECORD*.
+:    Add a Trust Anchor for *DOMAIN* with DS record data *DSRECORD*. This does not
+     replace the existing Trust Anchor for *DOMAIN* but appends the *DSRECORD*.
```

Comment:
This text comes across a bit vague. Maybe 'This adds the new TA to the existing TA set for the domain.' or the like